function
thrower
(
)
{
throw
new
Test262Error
(
)
;
}
assert
.
throws
(
Test262Error
function
(
)
{
let
{
[
thrower
(
)
]
:
x
}
=
{
}
;
}
)
;
reportCompare
(
0
0
)
;
